### PR TITLE
Update the instruction on authentication

### PR DESCRIPTION
**Pull request by HalynaDumych**
Add info on the token expiration time.

Relates-To: OLPEDGE-2391

Signed-off-by: Halyna Dumych <ext-halyna.dumych@here.com>

### CODE REVIEW

File: docs/authenticate.md [hunk 1]
```diff
@@ -40,18 +40,22 @@ You can use an access token to authenticate to the HERE platform and start worki
    import { UserAuth, requestToken } from "@here/olp-sdk-authentication";
    ```
 
-5. Create the `UserAuth` instance in one of the following ways:
+5. Create the `UserAuth` instance and, if needed, specify the token expiration time in one of the following ways:
 
-   - For token authentication, specify the `tokenRequester` method and your credentials.
+    > #### Note
+    > The default token expiration time must be equal to or more than zero. Ignored if it is zero or greater than the default expiration time supported by the access token endpoint. The default token expiration time is 3600 seconds (one hour).
```
Comment (OleksiiZubko) [inline docs/authenticate.md:46]: 24hrs

Comment (OleksiiZubko) [inline docs/authenticate.md:46]: in seconds

File: docs/authenticate.md [hunk 2]
```diff
@@ -40,18 +40,22 @@ You can use an access token to authenticate to the HERE platform and start worki
...
      ```typescript
      const userAuth = new UserAuth({
        tokenRequester: requestToken,
        credentials: credentials,
+       expiresIn?: number;
      });
      ```
 
-   - To authenticate with local authorization, specify the environment in which you work, your credentials, and the `tokenRequester` method.
+   - To authenticate with local authorization, specify the environment in which you work, your credentials, the `tokenRequester` method, and the token expiration time.
```
Comment (OleksiiZubko) [inline docs/authenticate.md:58]: it's optional parameter, I'm not sure if we need to add that there

File: docs/authenticate.md [hunk 3]
```diff
@@ -62,18 +66,20 @@ You can use an access token to authenticate to the HERE platform and start worki
        customUrl: "http://YourCustomEnvironment",
        credentials: credentials,
        tokenRequester: requestToken,
+       expiresIn?: number;
      });
      ```
 
-   - For project authentication, specify the `tokenRequester` method, your project name, and credentials.
+   - For project authentication, specify the `tokenRequester` method, your project name, credentials, and the token expiration time.
```
Comment (OleksiiZubko) [inline docs/authenticate.md:73]: the same, it's optional

Comment (HalynaDumych) [reply docs/authenticate.md:73]: Specified that it's optional in the note.

File: docs/authenticate.md (hunk 2 above)
Comment (HalynaDumych) [reply docs/authenticate.md:58]: Specified that it's optional in the note.

File: docs/authenticate.md (hunk 1 above)
Comment (HalynaDumych) [reply docs/authenticate.md]: Added

Comment (HalynaDumych) [reply docs/authenticate.md]: Changed.

**Review by OleksiiZubko — APPROVED** [review]

**Pull request merged**